DependencyInjection/HttplugExtension.php

```diff
@@ -27,6 +27,17 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('services.xml');
         $loader->load('plugins.xml');
         $loader->load('discovery.xml');
+
+        if ($config['toolbar']['enabled'] && $s = $container->hasParameter('kernel.debug') && $container->getParameter('kernel.debug')) {
```

Comment:
so here you would do something like:

$enabled = is_boolean($config['toolbar']['enabled']) ? $config['toolbar']['enabled'] : $container->hasParameter('kernel.debug') && $container->getParameter('kernel.debug');
if ($enabled) {
...